base_local_planner/src/obstacle_cost_function.cpp

```diff
@@ -42,10 +42,13 @@
 
 namespace base_local_planner {
 
-ObstacleCostFunction::ObstacleCostFunction(costmap_2d::Costmap2D* costmap) 
-    : costmap_(costmap), sum_scores_(false) {
+ObstacleCostFunction::ObstacleCostFunction(costmap_2d::Costmap2D* costmap)
+    : costmap_(costmap), max_trans_vel_(0.55)
+    , sum_scores_(false), max_scaling_factor_(0.2), scaling_speed_(0.25) {
   if (costmap != NULL) {
     world_model_ = new base_local_planner::CostmapModel(*costmap_);
+  } else {
+    world_model_ = NULL;
```

Comment:
It's better to use ```world_model_(NULL)```.

Reply:
Certainly initialize with NULL,
```new base_local_planner :: CostmapModel (* costmap_);```
It is good to set the value with.

Reply:
OK.